eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -49,40 +49,90 @@ param(
   [string]$PRBody = $PRTitle,
 
   [Parameter(Mandatory = $false)]
-  [string]$PRLabels
+  [string]$PRLabels,
+
+  [Parameter(Mandatory = $false)]
+  [string]$UserReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$TeamReviewers,
+
+  [Parameter(Mandatory = $false)]
+  [string]$Assignees,
+
+  $PRNumber
```

Comment:
Why are you adding a PRNumber here?

Reply:
That was just for testing. I will remove it.